configuration/cardano/mainnet-config.yaml

```diff
@@ -41,7 +41,11 @@ TargetNumberOfActivePeers: 20
 TargetNumberOfEstablishedPeers: 30
 TargetNumberOfKnownPeers: 150
 TargetNumberOfRootPeers: 60
-
```

Comment:
This block of new config should be removed -- it is already included below.
In the original block on the bottom, the `SyncTargetNumberOfEstablishedBigLedgerPeers` is set correctly to `40`.